src/libs/kata-types/Cargo.toml

```diff
@@ -34,8 +36,10 @@ safe-path = { path = "../safe-path" }
 [dev-dependencies]
 tempfile = "3"
 test-utils = { path = "../test-utils" }
-nix = "0.24.2"
+nix = "0.26"
```

Comment:
There is now a newer version of this crate.

Reply:
Bumped to 0.28.0